cookie_website/settings.py

```diff
@@ -139,3 +139,11 @@
 
 # Heroku settings
 django_heroku.settings(locals())
+
+# Email settings
```

Comment:
The website I had recommended adding the HOST/USER/PASSWORD to an environment file to obscure it, but I know you also need to do the secret key as well, so I just left these in here for now.